06_puppy/willshen8/mapStore.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+type MapStore map[uint32]*Puppy
+
+func (m MapStore) CreatePuppy(p Puppy) {
```

Comment:
what if puppy with `id` already exist?

Reply:
It will override the old value based on the current implementation. Not sure what should be the ideal scenario here (insert or ignore duplicate key). Perhaps this can be built into lab7 when we deal with errors?

Reply:
sounds good. 👍